lib/ProductOpener/Config2_docker.pm

```diff
@@ -158,4 +159,17 @@ $crm_username = $ENV{ODOO_CRM_USER};
 $crm_db = $ENV{ODOO_CRM_DB};
 $crm_pwd = $ENV{ODOO_CRM_PASSWORD};
 
+# Slack URLs
+%slack_hook_urls = ();
+
+if ((defined $ENV{SLACK_HOOK_URLS}) and ($ENV{SLACK_HOOK_URLS} ne '')) {
+	foreach my $kvp (split(',', $ENV{SLACK_HOOK_URLS})) {
+		if (not($kvp =~ s/^(?<channel>.+)=(?<url>https?.+)$//)) {
+			next;
+		}
+
+		$slack_hook_urls{$+{channel}} = $+{url};
+	}
+}
+
```

Comment:
Why put it there ? 

We have a lib/ProductOpener/ConfigEnv.pm for configurations that depends on environment, and it will be available everywhere.

Reply:
I wasn't aware of that file TBH. 😄

Reply:
Fixed in c7ffc6c29c905468c998a531a7afd3485cbbd645